src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) => {
+    count++;
+
+    return (y) => {
+      if (count < 4) {
+        return x + y;
+      } else {
+        return count % 2 === 0 ? 'Bzzz... Error!' : x + y;
+      }
+    };
```

Comment:
Don't nest ternary operator inside `if`. You need to simplify your logic. Return `x + y` in single place.